browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -17182,13 +17243,42 @@ index 0000000000000000000000000000000000000000..61d1f18d8b7ba5eda1ec1e3c7fd59f3d
 +        callback->sendSuccess();
 +    });
 +}
++
++void WebPageInspectorInputAgent::dispatchWheelEvent(int x, int y, std::optional<int>&& modifiers, std::optional<int>&& deltaX, std::optional<int>&& deltaY, Ref<DispatchWheelEventCallback>&& callback)
++{
++    OptionSet<WebEvent::Modifier> eventModifiers;
++    if (modifiers)
++        eventModifiers = eventModifiers.fromRaw(*modifiers);
++
++    float eventDeltaX = 0.0f;
++    if (deltaX)
++        eventDeltaX = *deltaX;
++    float eventDeltaY = 0.0f;
++    if (deltaY)
++        eventDeltaY = *deltaY;
++    m_wheelCallbacks->append(WTFMove(callback));
++
++    // Convert css coordinates to view coordinates (dip).
++    double totalScale = m_page.pageScaleFactor() * m_page.viewScaleFactor();
++    x = clampToInteger(roundf(x * totalScale));
```

Comment:
`clampToInteger(x * totalScale)` ?

Reply:
This is copied from our mouse dispatching code.